types/hashing.go

```diff
@@ -40,3 +41,14 @@ func (h *Header) Hash() Hash {
 func (b *Block) Hash() Hash {
 	return b.SignedHeader.Hash()
 }
+
+// Hash returns hash of the Data
+func (d *Data) Hash() (Hash, error) {
```

Comment:
i think we intend to have the da commitment (obtained using `blob.CreateCommitment` to be stored in DataHash instead of merkle hash, if i am not mistaken @nashqueue?

Reply:
Yes that is correct

Reply:
Created a follow up issue: https://github.com/rollkit/rollkit/issues/1203. Note that this issue is blocked on a `v1` priority issue.